config/mysql.go

```diff
@@ -12,12 +12,17 @@ type MySQL struct {
 	User     string `envconfig:"MYSQL_USER"`
 	Pw       string `envconfig:"MYSQL_PW"`
 	Host     string `envconfig:"MYSQL_HOST_NAME"`
+	Port     int    `envconfig:"MYSQL_PORT"`
 	DBName   string `envconfig:"MYSQL_DB_NAME"`
 	Location string `envconfig:"MYSQL_LOCATION"`
 }
 
-// DefaultLocation is used for MySQL connections.
-const DefaultLocation = "America/New_York"
+const (
+	// DefaultLocation is the default location for MySQL connections.
+	DefaultLocation = "America/New_York"
+	// DefaultPort is the default port for MySQL connections.
+	DefaultPort = 3306
```

Comment:
There are a few other configs in this package that have a port. Mind renaming this to `DefaultMySQLPort` to remove any possible confusion?